src/main/resources/configForReports.conf

```diff
@@ -2,4 +2,8 @@ activeUserReport_path=twitter-analytics/reporting/activeUsers
 
 activeUserDfReportPath=${activeUserReport_path}/dataFrameReport
 activeUserSqlReportPath=${activeUserReport_path}/sqlReport
-activeUserDsReportPath=${activeUserReport_path}/dataSetReport
\ No newline at end of file
+activeUserDsReportPath=${activeUserReport_path}/dataSetReport
+
+retentionPeriod=7
```

Comment:
the config is dedicated to spark job building report.
retention management doesn't get handled by the job, another/separate application is responsible for the procedure. another application - separate config.

Reply:
> reportDataRetention.sh

please you imperative naming.

Reply:
> Name of script updated to reportRetentionManager.sh

are you sure that it is imperative now?